integration_tests/models/actual/snowplow_unified_views_mobile_screen_engagement_actual.sql

```diff
@@ -0,0 +1,66 @@
+{#
+Copyright (c) 2024-present Snowplow Analytics Ltd. All rights reserved.
```

Comment:
Now we got to know that this has to be the year of the first release, should have been fine with 2023 then.

Reply:
Hmm, I think it should be different per file – so for each file the copyright is since it was created until present. Or did I get that wrong?

Reply:
I think it should match the package copyright as a whole - but also it makes it easier to find replace if we need to in the future...

Reply:
All right, have changed to `2024-present`.